contracts/KetherNFTRender.sol

```diff
@@ -21,26 +21,65 @@ contract KetherNFTRender is ITokenRenderer {
       '</g></svg>')));
   }
 
+  // Thanks to @townsendsam for giving us this reference https://gist.github.com/townsendsam/df2c420accb5ae786e856c97d13a2de6
```

Comment:
@townsendsam Thanks for sharing your reference, happy to add more here as a shoutout. Any requests?

Reply:
No, I don't have any particular request in mind. Happy to help!